src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -908,6 +911,10 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
                         ShowContinueError(state, "Infiltration Coefficients are all zero.  No Infiltration will be reported.");
                     }
                 }
+                if (!lAlphaFieldBlanks(5)) {
```

Comment:
Process new input

Reply:
Since #10295, and when using getObjectItem, you don't have to check if the field is blank if it has a default, even if the field is not present. I know it's a little scary to go there but the field will get stuffed with the default even after min fields (A5 is past min-fields = 12). 5ZoneEndUses - but any file will show this result. I'm not asking for a change here just pointing out a new field rule.

![image](https://github.com/user-attachments/assets/2ff823ca-38e0-4708-bae9-275ff4313273)